package.json

```diff
@@ -7,9 +7,9 @@
   "scripts": {
     "lint": "standard | snazzy",
     "lint:fix": "standard --fix",
-    "unit": "tap test/*.test.js",
-    "typescript": "tsd",
-    "test": "npm run lint && npm run unit && npm run typescript",
+    "test": "npm run test:unit && npm run test:typescript",
+    "test:typescript": "tsd",
+    "test:unit": "tap test/*.test.js",
```

Comment:
Can you check if .taprc contains 

```
files:
  - 'test/**/*.test.js
```

If not add it to the taprc s

Reply:
There wasn't a .taprc file, so I changed the path directly like in [multipart](https://github.com/fastify/fastify-multipart/blob/aced13c21f491b0d0b0b8324688cedf2db548e6e/package.json#L50)